### PR TITLE
enh(files_reminders): Allow clearing reminders

### LINKED ISSUE #41368: Allow clearing reminders on files
The clear reminder action should be available if a reminder exists

On 27:
image

Not sure about implementation as FileAction `enabled()` is not awaitable

Previously a request was made to the backend to check if a reminder was set https://github.com/nextcloud/server/blob/6250c7d3a9a9e754f7e7c90afc0267f1d88cddfb/apps/files_reminders/src/main.ts#L57

### PR DESCRIPTION
**Pull request by Pytal**
- Resolve https://github.com/nextcloud/server/issues/41368

## Summary

- Provide reminder as a dav attribute
- Allow clearing reminders (regression from migration to new FileAction API)

| Screenshot |
| --- |
| image |

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by susnux — APPROVED** [review]
Looks good but also add the property here:

https://github.com/nextcloud/documentation/blob/master/developer_manual/client_apis/WebDAV/basic.rst#supported-properties

File: apps/files_reminders/appinfo/info.xml [hunk 1]
```diff
@@ -29,4 +33,10 @@ Set file reminders.
 	<commands>
 		<command>OCA\FilesReminders\Command\ListCommand</command>
 	</commands>
+
+	<sabre>
+		<plugins>
+			<plugin>OCA\FilesReminders\Dav\PropFindPlugin</plugin>
+		</plugins>
+	</sabre>
```
Comment (skjnldsv) [inline apps/files_reminders/appinfo/info.xml:41]: I wonder if this is still the  clean way, like could we DI \OCA\DAV\Connector\Sabre\Server, and use the `addPlugin` directly? :thinking:

**Review by skjnldsv — APPROVED** [review]
Code looks good! :)

File: apps/files_reminders/appinfo/info.xml (hunk 1 above)
Comment (Pytal) [reply apps/files_reminders/appinfo/info.xml:41]: Will do that in a followup!

**Pull request merged**